lib/ui/view/channel/channel_view/channel_page_view.dart

```diff
@@ -245,7 +249,7 @@ Container channelInfo(String text, String nexttext) {
   );
 }
 
-Row row() {
+Row row(Function()? pressed) {
   final navigator = locator<NavigationService>();
```

Comment:
Kindly remove the declaration for the Navigation Service

Registering a service to the locator should come into your viewModel as your view should know nothing as to the logic